### PR TITLE
Task 3

### PR DESCRIPTION
**Pull request by Mateus-Resende**
- included and .sass-cache to gitignore
- created copy, compass, clean, jscs, newer and time grunt. 
- changed grunt function to es6
- included config in jshint to use es6

### CODE REVIEW

**Comment by Mateus-Resende** [comment]
Don't forget about me please @pcolenbr

File: Gruntfile.js [hunk 1]
```diff
@@ -1,7 +1,40 @@
...
 
   grunt.initConfig({
 
+    clean: {
+      build: {
+        src: ['.tmp/*']
+      }
+    },
+
+    compass: {
```
Comment (pcolenbr) [inline Gruntfile.js]: You compass is good for now.
But take sometime to look at some other configurations for the compass plugin

File: Gruntfile.js [hunk 2]
```diff
@@ -1,7 +1,40 @@
-module.exports = function(grunt) {
+
+module.exports = grunt => {
```
Comment (pcolenbr) [inline Gruntfile.js:1]: Nice touch using ECMA 6

File: Gruntfile.js [hunk 3]
```diff
@@ -1,7 +1,40 @@
...
+    clean: {
+      build: {
+        src: ['.tmp/*']
+      }
+    },
+
+    compass: {
+      dist: {
+        options: {
+          sassDir: 'styles',
```
Comment (pcolenbr) [inline Gruntfile.js]: To make sure you are getting all the css files use: styles/*_/_.[sass, scss]
Remember that *\* will include all folders, and that is important

**Pull request merged**